### DISCUSSION TITLE
Branch support

### DISCUSSION THREAD

**Discussion by ikunduraci** [lindell/multi-gitter#357] [Q&A]
Hi,

Does multi-gitter support running scripts in the context of every branch for a given repository?

Thanks

**Comment by lindell** [comment]
It does not. 

Could you explain what you are trying to do?

**Reply by ikunduraci** [reply]
I'd like to run multi-gitter across our organisation to check whether a particular file exists in each project, where a project could have any number of branches. If the file doesn't exist, it should create the file via an MR.
